luigi/parameter.py

```diff
@@ -223,6 +227,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
```

Comment:
May I suggest we start with defining `is_batchable` and `parse_list` as `_private`? Or do you already have use cases in mind where you want to override them?